src/main/java/net/neoforged/neoform/runtime/utils/OsUtil.java

```diff
@@ -1,7 +1,7 @@
 package net.neoforged.neoform.runtime.utils;
```

Comment:
Instead of having a separate class, I'd kinda just have `current` in OsType and all the `is` as instance methods.